src/ledger/tests.zig

```diff
@@ -330,12 +330,18 @@ pub const TestState = struct {
 };
 
 pub const TestDB = struct {
-    const dir = sig.TEST_STATE_DIR ++ "/blockstore";
+    const dir = sig.TEST_STATE_DIR ++ "blockstore";
```

Comment:
why remove the slash? I put this here because I try to make a habit of including the delimiter when constructing paths to depend on fewer assumptions.

Reply:
This was actually part of changes made while troubleshooting the issue to figure out what could be going wrong.

Returned in [11489a7](https://github.com/Syndica/sig/pull/396/commits/11489a75baa3471c069f493b15f6f630c2b5e65a)